packages/renderer/src/lib/preferences/docker-compat/PreferencesDockerCompatibilitySocketMappingStatus.svelte

```diff
@@ -16,7 +16,8 @@ let isLinux = $state(false);
 
 let dockerSocketMappingStatusInfo: DockerSocketMappingStatusInfo | undefined = $state(undefined);
 
-let engineType: 'kubernetes' | 'podman' | 'docker' | undefined = $state(undefined);
+let engineType: 'kubernetes' | 'podman' | 'docker' | 'lima' | undefined = $state(undefined);
+// 'lima' can provide any engine of 'kubernetes' or 'podman' or 'docker', either one or several
```

Comment:
suggestion (non-blocking): I think this comment would be better in the `packages/api/src/docker-compatibility-info.ts` ?